docs/guides/specify-runtime-options.mdx

```diff
@@ -0,0 +1,403 @@
+# This does bulk update.
+estimator.set_options(shots=4000)
+```
+  </TabItem>
+</Tabs>
+
+<span id="run-method"></span>
+### Run() method
+
+The run() method accepts options in V1 only.
+
+<Tabs>
+  <TabItem value="PrimV2" label="V2 primitives">
+In V2, the only values you can pass to `run()` are those defined in the interface.  That is, `shots` for Sampler and `precision` for Estimator. This overwrites any value set for `default_shots` or `default_precision` for the current run.
+
+```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+sampler = Sampler(mode=backend)
+# Default shots to use if not specified in run()
+sampler.options.default_shots = 500
+# Sample two circuits at 128 shots each.
+sampler.run([circuit1, circuit2], shots=128)
+
+# Sample two circuits with different numbers of shots.
+# 100 shots is used for circuit1 and 200 for circuit2.
+sampler.run([(circuit1, None, 100), (circuit2, None, 200)])
+```
+
+  </TabItem>
+
+  <TabItem value="PrimV1" label="V1 primitives">
+You can pass any options to `run()`. Because most users will only overwrite a few options at the job level, it is not necessary to specify the options category. The code below, for example, specifies `shots=1024` instead of `execution={"shots": 1024}` (which is also valid).
+```python
+estimator = Estimator(backend=backend, options=options)
+result = estimator.run(circuit, observable, shots=1024).result()
+print(f">>> Metadata: {result.metadata[0]}")
+```
+</TabItem>
+</Tabs>
+
+### Special cases (V2 only)
+
+#### Resilience level (Estimator only)
+
+The resilience level is not actually an option that directly impacts the primitive query, but specifies a base set of curated options to build off of. In general, level 0 turns off all error mitigation, level 1 turns on options for measurement error mitigation, and level 2 turns on options for gate and measurement error mitigation.
+
+Any options you manually specify in addition to the resilience level are applied on top of the base set of options defined by the resilience level. So in principle, you could set the resilience level to 1, but then turn off measurement mitigation, although this is not advised.
+
+In the following example, setting the resilience level to 0 initially turns off `zne_mitigation`, but `estimator.options.resilience.zne_mitigation = True` overrides the relevent setup from `estimator.options.resilience_level = 0`.
+
+``` python
+from qiskit_ibm_runtime import EstimatorV2, QiskitRuntimeService
+from qiskit import QuantumCircuit
+
+service = QiskitRuntimeService()
+backend = service.backend("ibm_auckland")
+
+estimator = EstimatorV2(backend)
+
+estimator.options.default_shots = 100
+estimator.options.resilience_level = 0
+estimator.options.resilience.zne_mitigation = True
+```
+
+### Shots (Sampler only)
```

Comment:
I think you meant to have this section (and the next) to be inside "Special cases (V2 only)"? Since both are also V2 only. 

```suggestion
#### Shots (Sampler only)
```